src/parser/jobs/ast/ArcanaSuggestions.js

```diff
@@ -122,6 +124,51 @@ export default class ArcanaSuggestions extends Module {
 		//					<Trans id="ast.arcana-suggestions.messages.header4">Remarks</Trans></Table.HeaderCell>
 		// {this.RenderRemark(artifact)}
 
+		const cardDisplayPanel = [{
```

Comment:
Not a blocking issue - just thinking. Currently, this is being used to show/collapse the entire table.

There's existing precedence to use the accordion with one expando per "entry", though I'm not sure how applicable it is to this case, given the table gives context to itself. Bit 50/50 really, I wonder if there's a better widget altogether?

Reply:
I considered making a UI that would show the Aracana Spread, and let them click "next" and "back". And the info would slide in and out. It would compress it nicely. But I still feel that there are people who would find it easier to just scroll through it. 
Know any widget I could use that would fit the bill? Like a carousel widget or something

How about that PLUS an additional option to "View logs in a list" like what we have here?